demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -69,6 +73,14 @@ def __new__(
         if content_type:
             content_type_to_model[content_type] = model_cls
             model_cls.content_type = content_type
+        # validates that if there are any lazy properties in the class model, add them as a class attribute
+        # so we would be able to load them during graph creation
```

Comment:
please add to docstring instead.
also for consistency let's use `model_cls` below

Reply:
what do you mean by "let's use model_cls below"?